### ISSUE TITLE
Transpile error when specifying target in typescriptOptions

### ISSUE THREAD

**Issue by unional** [systemjs/systemjs#608]
I try to set `target: "es3"` in the typescriptOptions, But when I do that, I got this error in the `plugin.js`

````
export function translate(load) {
   logger.debug(`systemjs translating ${load.address}`);
````

transpiled to:
````
    function translate(load) {
        logger.debug(`systemjs translating ${ + load.address + );  // missing closing ` and "}"
````
in `github:frankwallis/plugin-typescript@2.0.1/lib/plugin.js` (note that it is `.js` file so I suspect it is systemjs issue and not typescript or plugin-typescript.

In fact it doesn't matter if I get the target to "es5" or "es6".  As long as I set it, it is not working.

**Comment by guybedford** [comment]
//cc @frankwallis any ideas what might be happening here?

**Comment by frankwallis** [comment]
See https://github.com/Microsoft/TypeScript/issues/3976

The TypeScript transpile method does not currently accept the ```target``` option as a string. The built in TypeScript transpiler just passes the options straight through, so either it or the transpile method will need to be updated if passing this configuration as a string is to be supported.

The issue can be avoided by specifying the option as the actual enum value, and plugin-typescript has been updated to support this (it previously was not handling a value of 0 (es3) correctly).

I have also updated plugin-typescript to correctly convert the string to the correct enum value.
